std/os/windows/kernel32.zig

```diff
@@ -4,10 +4,10 @@ pub extern "kernel32" stdcallcc fn CancelIoEx(hFile: HANDLE, lpOverlapped: LPOVE
 
 pub extern "kernel32" stdcallcc fn CloseHandle(hObject: HANDLE) BOOL;
 
-pub extern "kernel32" stdcallcc fn CreateDirectoryW(lpPathName: [*]const u16, lpSecurityAttributes: ?*SECURITY_ATTRIBUTES) BOOL;
+pub extern "kernel32" stdcallcc fn CreateDirectoryW(lpPathName: LPCWSTR, lpSecurityAttributes: ?*SECURITY_ATTRIBUTES) BOOL;
```

Comment:
Can you explain your thoughts about these changes? I've been slowly converting code the other way.

Reply:
oh! i saw `LPCWSTR` is some places, but not others so i thought i would just make them all consistently use `LPCWSTR` and etc. i really don't have a preference because they same types.

Reply:
In that case, I have a slight preference for the naked types (how it was before), so if you don't mind could you leave out these changes from the PR?

Reply:
sure thing!